invenio_app_ils/ui/backoffice/src/common/api/user.js

```diff
@@ -0,0 +1,12 @@
+import { http } from './base';
+
+const userURL = '/users/';
+
+const getRecord = userPid => {
+  return http.get(`${userURL}${userPid}`);
+};
+
+export const user = {
+  url: userURL,
+  getRecord: getRecord,
```

Comment:
it is not a record, it might be misleading.
I would also change the other to have `get` only or `getLoan`, the react app doesn't care if it is a record... but I let @topless comment on this 😅

Reply:
@zzacharo true, it feels more intuitive. Should I change the other api calls for Loans and Items too ? @ntarocco

Reply:
@ntarocco the shorter the better, @zzacharo was faster on this one, that is the for I would use to get a user and every other model.